client/src/screens/AddItemScreen/AddItemScreen.tsx

```diff
@@ -43,50 +46,86 @@ export const AddItemScreen = ({navigation, route}: AddItemScreenProps) => {
             .catch(console.error);
     };
 
-    const handleSave = () => {
-        axiosInstance.post('/items/', {
-            name: itemName,
-            imageUrl: image,
-            categoryId,
-        })
-            .then(response => {
-                console.log(response.data);
-                Toast.show({
-                    type: 'success',
-                    text1: 'הפריט נשמר בהצלחה',
-                    text2: 'הפריט שהוספת נשמר במערכת ⭐️',
-                });
-                navigation.pop();
-            })
-            .catch(error => {
-                Toast.show({
-                    type: 'error',
-                    text1: 'השמירה נכשלה',
-                    text2: 'הפרטים שהזנת לא נשמרו במערכת ⛔️',
-                });
-                console.log(error);
+    const uploadImageToS3 = async () => {
+        const imageType = image.split('.')[1];
+        const imageName = `${Date.now()}.${imageType}`;
```

Comment:
I prefer imageName to be `s3-${email}-${Date.now()}.${imageType}`
Then in backend you'll filter by startsWith("s3-") instead of startsWith("https://")